goentri.com.prometheus.json

```diff
@@ -3,13 +3,14 @@
   "providerName": "Entri",
   "serviceId": "prometheus",
   "serviceName": "Prometheus",
-  "version": 1,
+  "version": 2,
   "logoUrl": "https://cdn.goentri.com/logo.svg",
   "description": "Allows user to easily set up domain using Entri",
   "syncPubKeyDomain": "goentri.com",
   "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com, app.goentri.com",
   "variableDescription": "customDomain1, customDomain2, customDomain3 and customDomain4 are the subdomain to point to websitePointsTo1 websitePointsTo2, websitePointsTo3, websitePointsTo4 respectively. hostValue and customValue is the dynamic host value assigned to the domain",
   "sharedProviderName": true,
+  "hostRequired":true,
```

Comment:
Sure about it?
I see no CNAME on APEX or alike.

Reply:
Hi Pawel, thanks for checking. I just set hostRequired as false.